clang/examples/Attribute/Attribute.cpp

```diff
@@ -52,7 +52,7 @@ struct ExampleAttrInfo : public ParsedAttrInfo {
   AttrHandling handleDeclAttribute(Sema &S, Decl *D,
                                    const ParsedAttr &Attr) const override {
     // Check if the decl is at file scope.
-    if (!D->getDeclContext()->isFileContext()) {
+    if (!D->getDeclContext()->isFileContext() && !isa<LabelDecl>(D)) {
```

Comment:
What is happening here?  I wouldn't expect a `LabelDecl` to appear at file scope anyway, it isn't really possible.

Reply:
Moot; I've removed the plugin-based testing.